clang/lib/Format/ContinuationIndenter.cpp

```diff
@@ -587,7 +590,7 @@ bool ContinuationIndenter::mustBreak(const LineState &State) {
       !State.Line->ReturnTypeWrapped &&
       // Don't break before a C# function when no break after return type.
       (!Style.isCSharp() ||
-       Style.AlwaysBreakAfterReturnType != FormatStyle::RTBS_None) &&
+       Style.AlwaysBreakAfterReturnType > FormatStyle::RTBS_AllowShortType) &&
```

Comment:
I know Owen proposed this. But I have to note, that I personally really don't like greater or lesser with enums.

Reply:
What I suggested is more consistent with the existing code in clang-format. Also, explicitly including/excluding every enum value is cumbersome and doesn't provide any real advantage, IMO.

Reply:
I know, just wanted to state my opinion. The cumbersome part could be extracted into a function. I just don't like treating enums as numbers in general. Sometimes there are exceptions, but in my opinion I would use this in clang-format only for `Standard`.